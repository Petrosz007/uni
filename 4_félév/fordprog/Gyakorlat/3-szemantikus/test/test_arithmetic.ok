program test_arithmetic
begin
    write(2+3)
    write(3-2)
    write(2*3)
    write(22/5)
    write(22%5)
    write(22<5)
    write(5<22)
    write(5>22)
    write(22>5)
    write(22>=5)
    write(5>=22)
    write(5<=22)
    write(22<=5)
    write(22<=22)
    write(22>=22)
    write(5=22)
    write(5=5)
    write(((3+2)*4/5+1)%2)
end
