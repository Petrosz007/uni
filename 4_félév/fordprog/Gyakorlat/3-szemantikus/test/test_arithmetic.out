5
1
6
4
2
false
true
false
true
true
false
true
false
true
true
false
true
1
